[12.0] [IMP] locomotive easy config

lmignon opened the pull request:
Add new button to easily configure locomotive from informations managed by odoo (api key, odoo url, index names, ....)

lmignon committed:
[IMP] shopinvader_locomotive: Add new button to easily configure locomotive from informations managed by odoo (api key, odoo url, index names, ....)

lmignon committed:
[ADD] setup.py

rousseldenis reviewed (approved):
Code review. Great!

Cedric-Pigeon reviewed (approved):
Awesome. Thanks.
Code review ok.

lmignon committed:
[IMP]shopinvader_demo_app:  Makes demo app data ready to work with docker-locomotive-shopinvader-demo

lmignon committed:
[IMP] shopinvader_assortment_demo: Improves demo data
Only export products into the 'Meuble' category
When using assortment, specify a category binding level to get the category automatically binded

lmignon commented:
/ocabot merge patch
